packages/app/src/app/pages/Dashboard/Content/routes/Shared/index.tsx

```diff
@@ -42,7 +45,25 @@ export const Shared = () => {
         showFilters
       />
 
-      <VariableGrid items={items} page={pageType} />
+      {items.length > 0 ? (
+        <VariableGrid items={items} page={pageType} />
+      ) : (
+        <EmptyPage.StyledWrapper>
+          <EmptyPage.StyledDescription as="p">
+            There are currently no sandboxes shared with you.
+          </EmptyPage.StyledDescription>
+          <EmptyPage.StyledGrid>
+            <ArticleCard
+              title="Get feedback in context"
+              thumbnail="/static/img/thumbnails/shared.png"
+              url="https://codesandbox.io/team"
+              onClick={() =>
+                track('Shared sandboxes - open docs from empty state')
+              }
+            />
+          </EmptyPage.StyledGrid>
+        </EmptyPage.StyledWrapper>
```

Comment:
Because it's less than 15 lines, I didn't think it was worth creating a separate file/component for it.